securedrop/source_templates/lookup.html

```diff
@@ -40,11 +40,15 @@ <h1 class="headline">{{ gettext('Submit Materials') }}</h1>
   <hr class="no-line">
   <div class="pull-right">
     <button type="submit" class="sd-button btn primary" id="submit-doc-button">
-      <img class="icon off-hover" src="{{ url_for('static', filename='i/font-awesome/cloud-upload-white.png') }}" width="20px" height="14px">
-      <img class="icon on-hover" src="{{ url_for('static', filename='i/font-awesome/cloud-upload-blue.png') }}" width="20px" height="14px">
+      <i class="fas fa-check-circle"></i>
       {{ gettext('SUBMIT') }}
     </button>
-    <a href="{{ url_for('main.lookup') }}" class="btn secondary" id="cancel">{{ gettext('CANCEL') }}</a>
+
+
+
+    <a href="{{ url_for('main.lookup') }}" class="btn secondary" id="cancel">
+      <i class="far fa-times-circle"></i> {{ gettext('CANCEL') }}
+    </a>
```

Comment:
I like this change, it feels intuitive to me: 

![screen shot 2018-06-18 at 9 55 47 am](https://user-images.githubusercontent.com/7832803/41550517-d8150dde-72dd-11e8-8913-edf933a272b9.png)

For the SecureDrop source interface, we want the web application to function nicely in Tor Browser with the security slider set to its safest state (which we recommend for source security). The thing is that FontAwesome SVGs actually don't display with this setting: 

![screen shot 2018-06-18 at 10 13 18 am](https://user-images.githubusercontent.com/7832803/41551292-3f518c46-72e0-11e8-9787-1c69467e868d.png)

That's the reason why we have PNGs for the other icons elsewhere on the source interface. So we'll want to replace these SVG icons with PNGs. I used to use [this site](http://fa2png.io/) to create icon PNGs of the appropriate color, but sadly it's down, but [this site](https://icons8.com/icon/set/cloud-upload/all) looks like it should do the trick. This is also why we have the `off-hover` and `on-hover` CSS classes: such that the image changes to the image with the appropriate color on hover. Btw, the functional tests [here](https://github.com/freedomofpress/securedrop/blob/develop/securedrop/tests/functional/source_navigation_steps.py#L35) are actually looking for the `off-hover` and `on-hover` CSS classes so once that is added for these new icons, the functional tests _should_ pass :metal:.